include/clap/helpers/plugin.hxx

```diff
@@ -888,6 +896,44 @@ namespace clap { namespace helpers {
       return applyConfigurationSuccess;
    }
 
+   //----------------------//
+   // clap_plugin_surround //
+   //----------------------//
+   template <MisbehaviourHandler h, CheckingLevel l>
+   bool Plugin<h, l>::clapSurroundIsChannelMaskSupported(const clap_plugin_t *plugin,
+                                                         uint64_t channel_mask) noexcept {
+      auto &self = from(plugin);
+      auto methodName = "clap_plugin_surround.is_channel_mask_supported";
+      self.ensureMainThread(methodName);
+      self.ensureIsInactive(methodName);
```

Comment:
This is incorrect, we can query the channel map even if the plugin is active.

Reply:
Corrected via 9a3baa7ca5c70f3486bd16e05903d324c8f56214